### DISCUSSION TITLE
Support for Service discovery via - Cloudmap

### DISCUSSION THREAD

**Discussion by kdhrubo** [awspring/spring-cloud-aws#855] [General]
Hi 
    There are several tickets (some closed some open) regarding support for AWS CloudMap for service discovery. It is unclear if the current 3. x release supports that from the document. It will be great if you can confirm if the feature is available in 3.x and if and when some documentation will be provided. 

Look forward to hearing from you.

**Comment by MatejNedic** [comment]
Hey @kdhrubo , currently 3.x does not support CloudMap. There is open PR for it https://github.com/awspring/spring-cloud-aws/pull/506 . We just don't have time to do a proper review since Tomaz, Maciej and I are working in our free time on this. Hopefully we will be able to get this up and running in future.
